wye/regions/forms.py

```diff
@@ -13,14 +13,16 @@ class Meta:
         exclude = ()
 
     def clean(self):
-        location = self.cleaned_data['location']
         error_message = []
-        for u in self.cleaned_data['leads']:
-            if not u.profile:
-                error_message.append('Profile for user %s not found' % (u))
-            elif u.profile.location != location:
-                error_message.append(
-                    "User %s doesn't belong to region %s" % (u, location))
+        if (self.cleaned_data.get('location', '') and
```

Comment:
Should we try by writing clean_locaiton() and clean_lead() to make it clean?
https://docs.djangoproject.com/en/1.9/ref/forms/validation/